tensorflow/lite/tools/flatbuffer_utils.py

```diff
@@ -274,6 +281,78 @@ def xxd_output_to_object(input_cc_file):
   return convert_bytearray_to_object(model_bytes)
 
 
+def byte_swap_buffer_content(buffer, chunksize, from_endiness, to_endiness):
+  """"Helper function for byte-swapping the buffers field.
```

Comment:
"""Helper function for byte-swapping the buffers field."""

1. make it one line
2. there are 4 double quotes.

Reply:
Updated. Thanks!